pkg/cli/debug/debug.go

```diff
@@ -157,7 +157,7 @@ func NewDebugOptions(streams genericclioptions.IOStreams) *DebugOptions {
 	return &DebugOptions{
 		PrintFlags:         genericclioptions.NewPrintFlags("").WithTypeSetter(scheme.Scheme),
 		IOStreams:          streams,
-		Timeout:            15 * time.Minute,
+		Timeout:            10 * time.Minute,
```

Comment:
I'd prefer we don't change this timeout, unless we have valid proof we should.

Reply:
ok, i have no idea why it's 15 min, but i'll change it back